core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/WorkspaceMigrationActor.scala

```diff
@@ -402,13 +410,148 @@ object WorkspaceMigrationMonitor {
       _ <- inTransaction { _ =>
         storageTransferJobs
           .map(job => (job.jobName, job.migrationId, job.destBucket, job.originBucket))
-          .insert((transferJob.getName, migration.id, destBucket.value, originBucket.value))
+          .insert((transferJob.getName, migrationId, destBucket.value, originBucket.value))
           .ignore
       }
 
     } yield transferJob
 
 
+  final def refreshTransferJobs: MigrateAction[(Long, Outcome)] =
+    for {
+      record <- peekTransferJob
+      (storageTransferService, billingProject) <- MigrateAction.asks { env =>
+        (env.storageTransferService, GoogleProject(env.billingProject.googleProjectId.value))
+      }
+
+      outcome <- MigrateAction.liftF {
+        OptionT {
+          storageTransferService
+            .listTransferOperations(record.jobName, billingProject)
+            .map(_.foldLeft(Option.empty[Outcome]) {
+              (outcome, operation) => outcome |+| getOperationOutcome(operation)
```

Comment:
What is the `|+|` operator? Googling seemed to not pick it up super well

Reply:
It's the semigroup combine operator

Reply:
Found another way of doing it :)